argusclient/client.py

```diff
@@ -591,6 +591,72 @@ def get_alerts_allinfo(self, ownerName=None, alertNameContains=None, shared=Fals
         """
         return self.argus._request("get", "alerts/allinfo", params=dict(ownername=ownerName, alertNameContains=alertNameContains, shared=shared, limit=limit))
 
+    '''
+    Functions for enabling composite alerts
+    '''
+
+    def get_composite_alert_children_info(self, alert_id):
+        if not alert_id: raise ValueError("Need to specify alert id")
+        uri_path = "alerts/{}/children/info".format(alert_id)
+        return self.argus._request("get", uri_path)
+
+    def create_composite_alert(self, payload):
+        if not payload: raise ValueError("Need to specify an payload")
+        alert_obj = self.argus._request("post", "alerts", dataObj=payload)
+        self._coll[alert_obj.id] = alert_obj
+        return alert_obj
+
+    def add_child_alert_to_composite_alert(self, alert_id, payload):
+        if not alert_id: raise ValueError("Need to specify composite alert id")
+        if not payload: raise ValueError("Need to specify  payload")
+        uri_path = "alerts/{}/children".format(alert_id)
+        alert_obj = self.argus._request("post", uri_path, dataObj=payload)
+        self._coll[alert_obj.id] = alert_obj
+        return alert_obj
+
+    def update_composite_alert(self, alert_id, payload):
+        if not alert_id: raise ValueError("Need to specify composite alert id")
+        if not payload: raise ValueError("Need to specify  payload")
+        uri_path = "alerts/{}".format(alert_id)
+        return self.argus._request("put", uri_path, dataObj=payload)
```

Comment:
Why doesn't this method store the alert in the _coll using ' self._coll[alert_obj.id] = alert_obj' like the other methods?

Reply:
Good catch. Added the code